components/activity_and_insights/insights/insights.tsx

```diff
@@ -24,13 +24,15 @@ import TopBoards from './top_boards/top_boards';
 
 import './../activity_and_insights.scss';
 
+type SelectOption = {
+    value: string;
+    label: string;
+}
+
 const Insights = () => {
     const dispatch = useDispatch();
-    const [filterType, setFilterType] = useState(InsightsScopes.MY);
-    const [timeFrame, setTimeFrame] = useState({
-        value: TimeFrames.INSIGHTS_7_DAYS,
-        label: localizeMessage('insights.timeFrame.mediumRange', 'Last 7 days'),
-    });
+    const [filterType, setFilterType] = useLocalStorageState('insightsScope', InsightsScopes.TEAM);
```

Comment:
Could we use `useGlobalState` for these instead of the new `useLocalStorageState`? I know the name is a bit confusing for that one, but it puts the state into redux-persist so that it's both accessible from redux and persisted as that name would imply.

Only 1/5 on that though. I'm not particularly married to redux-persist, but I would like to try to be consistent

Reply:
Good point, updated